src/coreclr/vm/codeman.cpp

```diff
@@ -1912,6 +1927,41 @@ void EEJitManager::SetCpuInfo()
     CPUCompileFlags.EnsureValidInstructionSetSupport();
 
     m_CPUCompileFlags = CPUCompileFlags;
+
+#if defined(TARGET_X86) || defined(TARGET_AMD64)
+    if (xarchCpuInfo.IsGenuineIntel)
+    {
+        // Some architectures can experience frequency throttling when executing
+        // executing 512-bit width instructions. To account for this we set the
+        // default preferred vector width to 256-bits in some scenarios. Power
+        // users can override this with `DOTNET_PreferredVectorBitWith=512` to
+        // allow using such instructions where hardware support is available.
+
+        if (xarchCpuInfo.FamilyId == 0x06)
```

Comment:
Where are we initializing `FamilyId`?

Reply:
Looks like this got lost on the original PR as part of a merge conflict resolution. Added it back.